webapp/urls.py

```diff
@@ -65,6 +65,12 @@
         {"slug": "juju", "template_path": "blog/topics/juju.html"},
         name="topic",
     ),
+    path(
+        r"blog/topics/robotics",
```

Comment:
is there a reason for this?

Reply:
Nope! TIL what `r` is. Removed `r` where it wasn't necessary.